_posts/2023-01-11-install-ruby-using-brew-rbenv.md

```diff
@@ -3,13 +3,18 @@ layout: post
 ---
 
 
-# First steps on new Ubuntu machine
+## First steps on new Ubuntu machine
```

Comment:
@mankec please put in README if you found some style guide that proposes two `##` instead of `#`